cmd/apps/inletsoperator_app_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) arkade author(s) 2021. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func Test_ProviderHetzner(t *testing.T) {
+	cmdInlets := MakeInstallInletsOperator()
+
+	// missing required flags
+	err := cmdInlets.ParseFlags([]string{"--provider", "hetzner"})
```

Comment:
I can't see the comment now, but I'm sure I advised not to test cobra itself, just the validation method we extracted.

I'll merge anyway to get this in.

Reply:
will remove the test